sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -438,3 +438,15 @@
   border: 1px solid $common-border-color !important;
   border-radius: 8px !important;
 }
+
+//Style added hide the header of data grid for student activity page.
+.student-activity-grid th {
+  border: none !important;
```

Comment:
is this only to remove the border and header? then `formio-datagrid-unset` itself should work, unless I am missing something
![image](https://user-images.githubusercontent.com/77353155/171068553-844d7e76-a982-4a9a-a4f7-3e9624fc5310.png)

Reply:
You are right. I missed to remove this part from .scss
My form uses formio-datagrid-unset.

student-activity-grid used just to left-pad the item